.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+### Description
+A few sentences describing the changes proposed in this pull request.
+
+### Status
+**Ready/Work in progress/Hold**
+
+### Types of changes
+<!--- Put an `x` in all the boxes that apply, and remove the not applicable items -->
+- [x] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
+- [ ] New tests added to cover the changes
+- [ ] Docstrings/Documentation updated
+
+
+## PR Checklist
+### PR Implementer
+This is a small checklist for the implementation details of this PR.
+
+If there are any questions regarding code style or other conventions check out our 
+[summary](https://github.com/gradslam/gradslam/blob/main/CONTRIBUTING.rst).
+
+- [ ] Did you discuss the functionality or any breaking changes before?
+- [ ] **Pass all tests**: did you test in local ? `pytest tests/`
+- [ ] Unittests: did you add tests for your new functionality?
+- [ ] Documentations: did you build documentation ? `cd docs/ && sphinx-build . _build`
+- [ ] Implementation: is your code well commented and follow conventions ? `black gradslam/ && flake8 gradslam/`
+- [ ] Docstrings & Typing: has your code documentation and typing ?
+- [ ] Update notebooks & documentation if necessary
+
+### gradslam Team
+<details>
+  <summary>gradslam team workflow</summary>
+  
+  - [ ] Assign correct label
```

Comment:
I would rename this to `- [ ] Triage` and introduce a new label that says `triaged` (i.e., all issues that have the `triaged` label have been assigned correct labels; this is useful when searching for unlabelled issues)

Reply:
Makes sense, modified